### ISSUE TITLE
DI injected IConfiguration cannot get any custom settings from local.settings.json

### ISSUE THREAD

**Issue by felinepc** [Azure/azure-functions-host#7225]
I'm using the just released  .NET 5 isolated access Azure Function project template for Visual Studio 2019, following[ this doc](https://docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-developer-howtos?tabs=browser&pivots=development-environment-vs).

I created a project using the `EventHubTrigger` template.

Per [this doc](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection), DI is available, and namely `Microsoft.Extensions.Configuration.IConfiguration` is available by default and safe to take as a dependency.

However, my injected `IConfiguration` instance fails to obtain any values in `local.settings.json`.

Contents of `local.settings.json`:
```
{
  "IsEncrypted": false,
  "Values": {
    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
    "EvhConnectionString": "omitted",
    "MyTestOption": "test:"
  }
}
```
Contents of Azure function:
```
public class MyAzureFunction
{
    private readonly IConfiguration _config;

    public MyAzureFunction(IConfiguration config)
    {
        _config = config;
    }

    [Function("MyAzureFunction")]
    public void Run([EventHubTrigger("acamsgraph", Connection = "EvhConnectionString")] string[] input, FunctionContext context)
    {
        Console.WriteLine($"Testing options :{_config.GetValue<string>("MyTestOption")}");
        var logger = context.GetLogger("MyAzureFunction");
        logger.LogInformation($"First Event Hubs triggered message: {input[0]}");
    }
}
```

Result: The string is null.

**Comment by v-anvari** [comment]
Hi @felinepc , As there are no templates for isolated projects, Can I know which template are you using in visual studio, Or are you using core tools to create and then debugging in visual studio. Can you run us through the repro steps that you are using
![image](https://user-images.githubusercontent.com/66122429/111167724-4cce2180-8599-11eb-8630-91c1639e840c.png)

**Comment by felinepc** [comment]
@v-anvari Yes I'm using the Azure Function Core Tools v3, as it's the only way to do this right now. I followed the exact [guide here](https://docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-developer-howtos?tabs=browser&pivots=development-environment-vs) , the only difference is choosing `EventHubTrigger` instead of `HttpTrigger`. So the steps are:

1. `func init LocalFunctionProj --worker-runtime dotnetisolated`
2. `func new --name HttpExample` (Then I chose the EventHubTrigger template as the command prompted me to)

So now it's a barebone function with just the codes I posted in my original post.

**Comment by v-anvari** [comment]
Hi @felinepc , Thankyou for the information, I am trying to repro and facing certain challenges as well. I shall revert back with an update

**Comment by felinepc** [comment]
@v-anvari Thanks. BTW, is there any ETA or issue tracker on when .NET 5 Azure Function will have a native/first-class experience in Visual Studio? Right now everything is done via command, especially the manually attached debugging, which is a pain. Also is deployment into a Linux-based app service hosting plan will be eventually supported?

**Comment by v-anvari** [comment]
Transferring this issue as it requires further expertise from .Net worker repo
